spring-integration-mail/src/test/java/org/springframework/integration/mail/ImapMailReceiverTests.java

```diff
@@ -199,7 +196,6 @@ public void testIdleWithMessageMapping() throws Exception {
 	}
 
 	@Test
-	@Disabled
```

Comment:
This test was disabled for a reason.
It is really not stable with connections on a slow environment like GitHub actions.